lib/ClangImporter/SwiftDeclSynthesizer.h

```diff
@@ -345,6 +345,8 @@ class SwiftDeclSynthesizer {
   /// function for the availability domain represented by `var`.
   FuncDecl *makeAvailabilityDomainPredicate(const clang::VarDecl *var);
 
+  bool isCGFloat(Type type);
```

Comment:
Nit: does this need to be a method of `SwiftDeclSynthesizer`? This only makes this method more cumbersome to call from, say, a `SimpleRequest` that does not have access to the synthesizer.